bench.js

```diff
@@ -30,28 +30,42 @@ var M = 10
   recomendation: return undefined
 */
 
-var buffer = new Buffer(8)
+var buffer = new Buffer.alloc(8)
```

Comment:
This should be `Buffer.alloc` instead of `new Buffer.alloc`, right?

Reply:
Yep, `new` is now removed.